ci-operator/config/openstack-k8s-operators/openstack-operator/openstack-k8s-operators-openstack-operator-main.yaml

```diff
@@ -104,7 +104,7 @@ tests:
     owner: openstack-k8s-operators
     product: ocp
     timeout: 1h20m0s
-    version: "4.10"
```

Comment:
For some reason we missed this test during migration from 4.10 to 4.12.
It should be the last one:
`openstack-k8s-operators]$ grep "4.10" -r . ` returns nothing